Teach_Breakout.py

```diff
@@ -0,0 +1,344 @@
+
+"""
+We make a DQN agent for playing Atari games, following (and modifying/extended) the tutorial here:
+https://becominghuman.ai/lets-build-an-atari-ai-part-1-dqn-df57e8ff3b26
+
+This code is specifically for Breakout.
+"""
+
+import gym
+import random
+import numpy as np
+import time
+import os
+from tensorflow import keras
+import keras
+from keras import layers
+from keras.models import Model
+
+
+# Make the model
+
+ATARI_SHAPE = (105, 80, 4)  # input image size to model (but note that we actually have to input shape: (batch,105,80,4))
```

Comment:
Can you not get this from `env.observation_space.shape`? Then it will generalize to other video games, and work if you transform the observation space.